api/proto/teleport/notifications/v1/notifications_service.proto

```diff
@@ -31,31 +31,26 @@ service NotificationService {
   rpc CreateUserNotification(CreateUserNotificationRequest) returns (Notification);
   // DeleteUserNotification deletes a user-specific notification.
   rpc DeleteUserNotification(DeleteUserNotificationRequest) returns (google.protobuf.Empty);
-  // ListUserNotifications returns a page of user-specific notifications.
-  rpc ListUserNotifications(ListUserNotificationsRequest) returns (ListUserNotificationsResponse);
 
   // CreateGlobalNotification creates a global notification.
   rpc CreateGlobalNotification(CreateGlobalNotificationRequest) returns (GlobalNotification);
   // DeleteGlobalNotification deletes a global notification.
   rpc DeleteGlobalNotification(DeleteGlobalNotificationRequest) returns (google.protobuf.Empty);
-  // ListGlobalNotifications returns a page of global notifications.
-  rpc ListGlobalNotifications(ListGlobalNotificationsRequest) returns (ListGlobalNotificationsResponse);
```

Comment:
I realized that there is no scenario where we want to only list global notifications, or only list user-specific notifications. We will always want to fetch from both datasets for a user.